packages/dropdowns/src/styled/menu/StyledMenu.tsx

```diff
@@ -167,13 +167,19 @@ interface IStyledMenuWrapperProps {
   isAnimated?: boolean;
 }
 
+/**
+ * 1. PopperJS requires a non-zero font-size to perform
+ * its initial placement correctly. This font-size value
+ * affects the content of the arrow styling ::before/::after
+ * elements.
```

Comment:
This second part isn't correct. The `font-size` has nothing to do with the arrow. It is simply pixel-sized now. See https://github.com/zendeskgarden/react-components/blob/next/packages/theming/src/utils/arrowStyles.ts for details.

Reply:
I've updated this comment.